api-inference-community/docker_images/sentence_transformers/Dockerfile

```diff
@@ -5,6 +5,7 @@ LABEL maintainer="Omar <omar@huggingface.co>"
 # RUN apt-get update -y && apt-get install libXXX -y
 
 COPY ./requirements.txt /app
+RUN pip install --no-cache-dir torch==1.9
```

Comment:
Why not have this in requirements?

Reply:
It's better for cache.

Having it before `requirements.txt` means as long torch version is not required to be updated in `requirements.txt` then you will hit cache and you won't have to redownload torch all the time.

Reply:
Sounds good, thanks!